packages/ui/resolution/ResolutionDecorator.js

```diff
@@ -28,6 +28,29 @@ const defaultConfig = {
 	 */
 	dynamic: true,
 
+	/**
+	 *
+	 * Determines how to get the best match screen type of current resolution. When set to `true`, the screen type is the one closest to the screen resolution without going over.
```

Comment:
Let's rephrase "without going over" and "without going below".
I guess smaller / greater would be fine.

Reply:
It's fixed